pkg/cluster/arooperator.go

```diff
@@ -9,16 +9,16 @@ import (
 	"github.com/Azure/ARO-RP/pkg/operator/deploy"
 )
 
-func (i *manager) ensureAROOperator(ctx context.Context) error {
-	dep, err := deploy.New(i.log, i.env, i.doc.OpenShiftCluster, i.kubernetescli, i.extcli, i.arocli)
+func (m *manager) ensureAROOperator(ctx context.Context) error {
```

Comment:
`i` was a vestige of the old `install` package - since it's now a `*manager` instead of an `*Installer`, it makes more sense to name the variable `m`.